src/locale/locales/de/messages.po

```diff
@@ -1858,43 +1796,40 @@ msgstr "E-Mail senden"
 msgid "Send Email"
 msgstr "E-Mail senden"
 
-#: src/view/shell/Drawer.tsx:295
-#: src/view/shell/Drawer.tsx:316
+#: src/view/shell/Drawer.tsx:295 src/view/shell/Drawer.tsx:316
 msgid "Send feedback"
-msgstr "Rückmeldung senden"
+msgstr "Feedback senden"
 
 #: src/view/com/modals/report/SendReportButton.tsx:45
 msgid "Send Report"
-msgstr "Bericht senden"
+msgstr ""
```

Comment:
"Meldung senden" or "Report senden", translation of "report" should be the same in other places https://github.com/bluesky-social/social-app/pull/2319/files#diff-aa13b140e1af15f6849455c56c65ceea70cfd95641707c46347b9c615e0462c5R106

Reply:
`"Meldung senden"` or just `"Melden"` seem appropriate.